From d9c9cc333e8a67749c20b3b30129f4d2297a3e0a Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston <jeremyhu@apple.com>
Date: Mon, 31 Oct 2011 03:34:47 +0000
Subject: Build fix for older servers (error: conflicting types for 'pciTag')

/usr/include/xorg/xf86Pci.h:254:25: note: previous declaration of 'pciTag' was here

Reported-by: Johannes Obermayr <johannesobermayr@gmx.de>
Signed-off-by: Jeremy Huddleston <jeremyhu@apple.com>
---
diff --git a/src/cir_pcirename.h b/src/cir_pcirename.h
index fa60a19..669b8fe 100644
--- a/src/cir_pcirename.h
+++ b/src/cir_pcirename.h
@@ -34,6 +34,10 @@ enum region_type {
     REGION_IO 
 };
 
+#include "xf86Module.h"
+
+#if GET_ABI_MAJOR(ABI_VIDEODRV_VERSION) >= 12
+
 #if (defined(__alpha__) || defined(__ia64__)) && defined (linux)
 #define PCI_DOM_MASK    0x01fful
 #else
@@ -55,6 +59,7 @@ pciTag(int busnum, int devnum, int funcnum)
 
 	return tag;
 }
+#endif /* GET_ABI_MAJOR(ABI_VIDEODRV_VERSION) >= 12 */
 
 #ifndef XSERVER_LIBPCIACCESS
 
--
cgit v0.9.0.2-2-gbebe
